Bottom: f5d1c8819a2c4f7cd164aca65934ec3d61252d5e
Top:    e987e9fb6e3d36821161cad967bb084a781f0bea
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-26 16:12:07 +0100

Fix outline drawing to only apply on copper layers

This matches comments in the code, and fixes (removes) outline drawing
on the solder mask prints.

This partially reverts commit 294494e56695251e63c0da9f243d6f59447f1c9f

"Fix PS rect/outline logic"


---

diff --git a/src/hid/ps/ps.c b/src/hid/ps/ps.c
index dbfc19e..408f225 100644
--- a/src/hid/ps/ps.c
+++ b/src/hid/ps/ps.c
@@ -845,10 +845,11 @@ ps_set_layer (const char *name, int group, int empty)
      we want to "print outlines", and we have an outline layer,
      print the outline layer on this layer also.  */
   if (global.outline &&
+      global.is_copper &&
       global.outline_layer != NULL &&
       global.outline_layer != PCB->Data->Layer+idx &&
-      strcmp (name, "outline") != 0 &&
-      strcmp (name, "route") != 0
+      strcmp (name, "outline") &&
+      strcmp (name, "route")
       )
     {
       dapi->draw_layer (global.outline_layer, &global.region, NULL);
